dashboard/app/models/artifact.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: artifacts
+#
+#  id                 :integer          not null, primary key
+#  name               :string(255)
+#  description        :text(65535)
+#  learning_module_id :integer
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+# Indexes
+#
+#  index_artifacts_on_learning_module_id  (learning_module_id)
+#
+
+class Artifact < ActiveRecord::Base
+  belongs_to :learning_module
+  has_many :artifact_assignments, class_name: 'ArtifactAssignment', dependent: :destroy
+
+  def self.create_from_params(learning_module, name, description)
+    artifact = Artifact.new
+    artifact.name = name
```

Comment:
Nit:  If we decide we need this custom create method, assign the fields in the order of the parameter names.  This make it easier to read and verify that all have been assigned.